cocotb/binary.py

```diff
@@ -64,22 +64,22 @@ def _clog2(val):
         exp += 1
 
 
-class BinaryRepresentation():
-    UNSIGNED         = 0  # noqa
-    SIGNED_MAGNITUDE = 1  # noqa
-    TWOS_COMPLEMENT  = 2  # noqa
+class BinaryRepresentation():  # noqa
+    UNSIGNED         = 0  #: Unsigned format
+    SIGNED_MAGNITUDE = 1  #: Sign and magnitude format
+    TWOS_COMPLEMENT  = 2  #: Two's complement format
 
 
 class BinaryValue(object):
     """Representation of values in binary format.
 
     The underlying value can be set or accessed using these aliasing attributes:
 
-        - `BinaryValue.integer` is an integer
-        - `BinaryValue.signed_integer` is a signed integer
-        - `BinaryValue.binstr` is a string of "01xXzZ"
-        - `BinaryValue.buff` is a binary buffer of bytes
-        - `BinaryValue.value` is an integer **deprecated**
+        - ``BinaryValue.integer`` is an integer
+        - ``BinaryValue.signed_integer`` is a signed integer
+        - ``BinaryValue.binstr`` is a string of "01xXzZ"
+        - ``BinaryValue.buff`` is a binary buffer of bytes
+        - ``BinaryValue.value`` is an integer **deprecated**
```

Comment:
Reviews in the previous PR changed this to using only a single ` IIRC. What's the motivation of changing it back?

Reply:
With a single back tick it just italicizes them, but with double it makes them monospaced/code-like.  I think monospaced is appropriate as they are surrounding a type, but that's just my opinion.

Reply:
From what I've seen, using single back-ticks results in a link being created, at least on other projects I've used sphinx on in the past.

Perhaps you need ``:py:`BinaryValue.value` `` for that to work.

Reply:
Actually, I needed `` :attr:`BinaryValue.integer` `` (short for `` :py:attr:`BinaryValue.integer` ``) but now it's hyperlinked, yay!